sync/datahub/metrichub_glossary.py

```diff
@@ -90,6 +93,20 @@ def _build_product_dict(product: str, metrics: List[MetricHubDefinition]) -> Dic
     }
 
 
+def _connect_terms_to_source_table(metrics: List[MetricHubDefinition]) -> None:
```

Comment:
Not `None` if it's gonna return the dictionary. You could also just write the yaml directly in this function too
```suggestion
def _connect_terms_to_source_table(metrics: List[MetricHubDefinition]) -> Dict[...]:
```